doc/en/customize.rst

```diff
@@ -97,6 +97,19 @@ check for ini-files as follows::
 .. _`how to change command line options defaults`:
 .. _`adding default options`:
 
+Change which files are added to sys.path
+-----------------------------------------------
+
+If you place a conftest.py file in the root directory of your project
+(as determined by pytest, see above.) pytest will run tests against 
+the code below that directory by adding it to your sys.path instead of 
```

Comment:
> As a newb coming from using nose's sys.path hackery

What if we mention this detail in `nose.rst`? This way users coming from `nose` will have an obvious place to understand the difference regarding how pytest finds the source code. We should instruct the user to create a virtual environment though, not mention about the sys.path implementation detail.